pkg/daemon/ceph/client/mgr.go

```diff
@@ -65,14 +70,25 @@ func MgrSetConfig(context *clusterd.Context, clusterName, mgrName string, cephVe
 }
 
 func enableModule(context *clusterd.Context, clusterName, name string, force bool, action string) error {
+
+	retryCount := 5
 	args := []string{"mgr", "module", action, name}
 	if force {
 		args = append(args, "--force")
 	}
-	_, err := NewCephCommand(context, clusterName, args).Run()
-	if err != nil {
-		return errors.Wrapf(err, "failed to mgr module enable for %s", name)
-	}
 
+	for i := 0; i < retryCount; i++ {
+		_, err := NewCephCommand(context, clusterName, args).Run()
+		if err != nil {
+			if i < retryCount-1 {
+				logger.Infof("failed to enable mgr module for %s. trying again...", name)
```

Comment:
```suggestion
				logger.Infof("failed to enable mgr module %q. trying again...", name)
```

Reply:
Moved retry loop to `MgrEnableModule` and added unit test for `enableModule`.